WireAPI/Sources/WireAPI/Public/Models/BackendInfo.swift

```diff
@@ -0,0 +1,41 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+/// Metadata about the local backend.
+
+public struct BackendInfo: Equatable {
```

Comment:
Does it make sense to move this into a swift package `WireAPITypes` (or similar name) and share code? 
Import in `WireTransport` and `WireAPI`.

Reply:
I think there would only be 2 or so models that would be shared by `WireTransport` and `WireAPI`, which is the `APIVersion` and `BackendInfo` (though this last one isn't quite the same). I don't see much benefit in adding a new package just for these two models at the moment, but certainly if there is more code to share then we could add a new package.